colin-api/src/colin_api/models/filing.py

```diff
@@ -113,16 +124,26 @@ class FilingSource(Enum):
         },
         'alteration': {
             'type_code_list': ['NOABE', 'NOALE', 'NOALR', 'NOALD', 'NOALA', 'NOALB', 'NOALU', 'NOALC'],
-            Business.TypeCodes.BCOMP.value: 'NOABE',
+            Business.TypeCodes.BCOMP.value: 'NOABE',  # No corp type change
+            Business.TypeCodes.BC_COMP.value: 'NOALA',  # No corp type change
+            Business.TypeCodes.ULC_COMP.value: 'NOALA',  # No corp type change
+            Business.TypeCodes.CCC_COMP.value: 'NOALA',  # No corp type change
+            Business.TypeCodes.BCOMP_CONTINUE_IN.value: 'NOABE',  # No corp type change
+            Business.TypeCodes.CONTINUE_IN.value: 'NOALA',  # No corp type change
+            Business.TypeCodes.ULC_CONTINUE_IN.value: 'NOALA',  # No corp type change
+            Business.TypeCodes.CCC_CONTINUE_IN.value: 'NOALA',  # No corp type change
             'BC_TO_BEN': 'NOALE',
             'BEN_TO_BC': 'NOALR',
             'ULC_TO_BEN': 'NOALD',
-            Business.TypeCodes.BC_COMP.value: 'NOALA',
-            Business.TypeCodes.ULC_COMP.value: 'NOALA',
-            Business.TypeCodes.CCC_COMP.value: 'NOALA',
             'ULC_TO_BC': 'NOALB',
             'BC_TO_ULC': 'NOALU',
             'BC_TO_CC': 'NOALC',
+            'C_TO_CBEN': 'NOALE',
+            'CBEN_TO_C': 'NOALR',
+            'CUL_TO_CBEN': 'NOALD',
+            'CUL_TO_C': 'NOALB',
+            'C_TO_CUL': 'NOALU',
+            'C_TO_CCC': 'NOALC',
```

Comment:
This may change according to outcome from colin test

Reply:
Well, for now it matches the non-continued-in alterations, so that's good.